src/main/java/com/synopsys/integration/jira/common/cloud/service/IssueService.java

```diff
@@ -154,13 +155,9 @@ public TransitionsResponseModel getTransitions(String issueIdOrKey) throws Integ
         return jiraCloudService.get(request, TransitionsResponseModel.class);
     }
 
-    public void addComment(IssueCommentRequestModel requestModel) throws IntegrationException {
+    public IssueCommentResponseModel addComment(IssueCommentRequestModel requestModel) throws IntegrationException {
         HttpUrl commentsUri = createApiCommentsUri(requestModel.getIssueIdOrKey());
-        JiraResponse response = jiraCloudService.post(requestModel, commentsUri);
-
-        if (response.isStatusCodeError()) {
-            throw new IntegrationException(String.format("Error commenting on issue; cause: (%d) - %s", response.getStatusCode(), response.getStatusMessage()));
-        }
+        return jiraCloudService.post(requestModel, commentsUri, IssueCommentResponseModel.class);
```

Comment:
I assume the error handling is now included in the post?

Reply:
Correct, all our clients now throw rest exceptions when an error status is received. I believe that is a reasonable assumption for a library service. (We did that for almost all error status's originally anyway)